modular_sand/code/datums/interactions/lewd_definitions.dm

```diff
@@ -65,10 +66,11 @@
 
 /mob/living/proc/add_lust(add)
 	var/cur = get_lust() //GetLust handles per-time lust loss
-	if((cur + add) < 0) //in case we retract lust
+	if((cur + add) < 0) //in case we retract lust, doesn't have to account for hair trigger, since we aren't multiplying with a negative
 		lust = 0
 	else
-		lust = cur + add
+		lust = cur + add * hair_trigger_mul
+
```

Comment:
This can be made into a modular edit by just modifying the `add` variable by the multiplier before it calls `. = ..()`